lib/js/atoms/App/App.vue

```diff
@@ -13,13 +13,31 @@ select {
 <script lang="ts">
 import Vuetify, { VApp } from 'vuetify/lib';
 import Vue from 'vue';
+
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome';
+import { fas } from '@fortawesome/pro-solid-svg-icons';
+import { far } from '@fortawesome/pro-regular-svg-icons';
```

Comment:
these lines will import all Icons from the lib and include them in the prod build.

Maybe we should initialize icons like in `fontawesome.ts`?